core/src/main/java/com/netflix/conductor/core/events/queue/dyno/DynoObservableQueue.java

```diff
@@ -121,7 +123,7 @@ public String getURI() {
     @VisibleForTesting
     private OnSubscribe<Message> getOnSubscribe() {
         return subscriber -> {
-            Observable<Long> interval = Observable.interval(pollTimeInMS, TimeUnit.MILLISECONDS);
+            Observable<Long> interval = Observable.interval(pollTimeInMS, TimeUnit.MILLISECONDS, Schedulers.io());
```

Comment:
We need to be careful here. io scheduler uses an unbounded thread pool. It doesn't seem like we're doing any IO here. We may end up incurring a lot of thread creation and context switching overhead here.

Reply:
As per official doc, ComputationScheduler should be used for small  cpu involving task and bound to the number of CPUs and should not include any I/O. The intended change is to use `ExecutorScheduler` to pass a separate thread pool with a configurable number of threads for i/o tasks. We need this change only if we need to handle large events and possibly help in queue pile up cases. 

Additional reference doc: https://www.aanandshekharroy.com/articles/2018-01/rxjava-schedulers

Reply:
In that case we should be using the `ExecutorScheduler` as opposed to the io scheduler here.